### PR TITLE
[W6.1e][W13-B1] Aadyaa Maddi

### PR DESCRIPTION
**Pull request by amad-person**
LO-JUnit:
Added a JUnit test to UtilsTest.java to test the isAnyNull(Object...) method in Utils.java.

### CODE REVIEW

File: test/java/seedu/addressbook/common/UtilsTest.java [hunk 1]
```diff
@@ -10,6 +10,22 @@
 
 public class UtilsTest {
 
+    @Test
+    public void isAnyNull() {
```
Comment (okkhoy) [inline test/java/seedu/addressbook/common/UtilsTest.java:14]: This is a correctly written test. 
Please also be aware of the 3 part test name format that may be useful when you write tests for your project.

**Review by okkhoy — COMMENTED** [review]
Please update the title of the PR to reflect your phase B team name.

**Comment by okkhoy** [comment]
@amad-person 
Some comments added. Please ack & close the PR after reading comments.

**Comment by amad-person** [comment]
Acknowledged, thanks!

**Pull request closed without merging**